### ISSUE TITLE
Stardust UI no longer exists

### ISSUE THREAD

**Issue by lukewarlow** [openui/open-ui#794]
Stardust UI no longer exists by that name. Most links to https://stardust-ui.github.io/react redirect to https://fluentsite.z22.web.core.windows.net/0.66.2 which itself says is deprecated and (brokenly) links to https://react.fluentui.dev/?path=/docs/concepts-introduction--page

StardustUI should be updated to the new React FluentUI stuff accounting for any changes.

**Comment by lukewarlow** [comment]
More granular but same overall theme to https://github.com/openui/open-ui/issues/758

**Comment by bkardell** [comment]
Yeah, there are several in here with these issues - I'm unsure what the right thing to do with them is because they are in research too and the research is still relevant, I think.

**Comment by gregwhitworth** [comment]
I agree with @bkardell on this one. We need some thought in how to retain the research while ensuring the matrix is up to date.
